steps/infisical-secrets/0.0.2/step.yml

```diff
@@ -0,0 +1,65 @@
+title: Infisical Secrets
+summary: |
+  Pull secrets from an Infisical secrets repository
+description: |
+  Use this step to access your cloud or self hosted Infisical secrets and inject them into your build pipeline.
+website: https://github.com/CodeFishStudio/bitrise-step-infisical-secrets
+source_code_url: https://github.com/CodeFishStudio/bitrise-step-infisical-secrets
+support_url: https://github.com/CodeFishStudio/bitrise-step-infisical-secrets/issues
+published_at: 2024-08-21T16:31:30.683254+09:30
+source:
+  git: https://github.com/CodeFishStudio/infisical-bitrise-step.git
+  commit: ca5df8db8b033c5f7c0d77f3c37c68ae7b7e436a
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/CodeFishStudio/bitrise-step-infisical-secrets
+inputs:
+- infisical_client: null
+  opts:
+    description: The client ID is required to identify the machine identity from Infisical
+    is_expand: true
+    is_required: true
+    summary: The client ID from infisical.
+    title: Infisical Client ID
+- infisical_client_secret: null
+  opts:
+    description: The client secret is required to authenticate the machine identity
+      from Infisical
+    is_expand: true
+    is_required: true
```

Comment:
Sensitive inputs should be marked as `is_sensitive`: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#sensitive-inputs